tardis/io/tests/test_model_reader.py

```diff
@@ -0,0 +1,35 @@
+import tardis
+from tardis.io.model_reader import (read_artis_density,
+read_simple_ascii_abundances)
+
```

Comment:
Yeah you're right! I just changed it. Going through to make sure all the lines are within the limit now.